app/controllers/health/sdh_case_management_notes_controller.rb

```diff
@@ -148,6 +148,28 @@ def display_note_object
     end
 
     private def permitted_request_params
+      versioned_activities_attributes = [].tap do |arr|
+        [
+          :id,
+          :date_of_activity,
+          :follow_up,
+          :_destroy,
+        ].each { |k| arr << k } # << so that the tap works
```

Comment:
Couldn't this be written as:
```suggestion
      versioned_activities_attributes = [
          :id,
          :date_of_activity,
          :follow_up,
          :_destroy,
        ]..tap do |arr|
        
```
or
```suggestion
      versioned_activities_attributes = [].tap do |arr|
        array += [
          :id,
          :date_of_activity,
          :follow_up,
          :_destroy,
        ]
```
This syntax feels really weird.

Reply:
Sure -- your first suggestion works, and I'll use it... The second one doesn't works: `a += b` is equivalent to `a = a + b`, so it creates a new array, instead of updating the tapped one (that is what the comment `# << so that the tap works` was trying to say!)